docs/Basic-Guides.md

```diff
@@ -0,0 +1,142 @@
+# Basic Guides
```

Comment:
This title isn't very descriptive. Also, why is this plural?

Reply:
What kind of title do you think would be better? I'll remove the "s".